tests/fallback/fallback_test.go

```diff
@@ -9,12 +9,23 @@ import (
 
 var _ = Describe("cOS booting fallback tests", func() {
 	var s *sut.SUT
+
+	BeforeSuite(func() {
+		sut.SnapshotVagrant()
+	})
+
+	AfterSuite(func() {
+		sut.SnapshotVagrantDelete()
+	})
+
 	BeforeEach(func() {
 		s = sut.NewSUT()
 		s.EventuallyConnects()
 	})
 	AfterEach(func() {
-		s.Reset()
+		if CurrentGinkgoTestDescription().Failed == false {
```

Comment:
mmh not sure if this is going to work, I'm pretty sure the AfterEach Reset here is required to have a pristine state on each `It` block, but I could recall wrongly

Reply:
for this to work we probably have to drop the retry, so when we fail we fail hard and we leave the vm so it can be inspected.

Otherwise, if the test was a success this will reset it to its original state and other tests can continue

Reply:
retry also dropped!